spec/beaker/host/unix/file_spec.rb

```diff
@@ -133,6 +133,14 @@ def to_s
         expect(filename).to be === correct
       end
 
+      it 'builds the filename correctly for amazon-based platforms' do
+        @platform = 'amazon-2023-x86_64'
```

Comment:
can this be made date independant?

Reply:
No, the 2023 is the release version. Sort of like how Ubuntu has it.